### PR TITLE
fix(auth): FR-1561 - changed back button for MFA recovery page

### PR DESCRIPTION
**Pull request by AntonReact**

### CODE REVIEW

**Comment by AntonReact** [comment]
https://frontegg.atlassian.net/secure/RapidBoard.jspa?rapidView=3&modal=detail&selectedIssue=FR-1561&assignee=5da57de11094ed0c44fe7825

File: packages/auth/src/Login/Login.tsx [hunk 1]
```diff
@@ -51,9 +51,22 @@ const stateMapper = ({ isLoading, isAuthenticated, loginState, onRedirectTo, rou
 });
 export const Login: FC<LoginProps> = (props) => {
   const Dynamic = useDynamicComponents(defaultComponents, props);
-  const { isLoading, isAuthenticated, step, onRedirectTo, routes, resetLoginState } = useAuth(stateMapper);
+  const { isLoading, isAuthenticated, step, onRedirectTo, routes, resetLoginState, setLoginState } = useAuth(
+    stateMapper
+  );
   const { t } = useT();
 
+  const backButtonProps = {
```
Comment (serslon) [inline packages/auth/src/Login/Login.tsx:59]: Much better will be using the **useMemo** hook for this property.

**Comment by AntonReact** [comment]
https://user-images.githubusercontent.com/54889005/105848258-66b2a580-5fe7-11eb-89c1-411ccb19073c.mov

**Comment by frontegg-david** [comment]
@aviadmizrachi I think the button in recovery page "Back to token screen" is not good. Can you please suggest other text?

**Comment by frontegg-david** [comment]
@AntonReact please check why the test checks have failed

**Pull request closed without merging**